letslambda.json

```diff
@@ -58,7 +145,12 @@
         "LambdaExecutionRole": {
             "Type": "AWS::IAM::Role",
             "Properties": {
-                "ManagedPolicyArns": [ {"Ref": "LetsLambdaManagedPolicy"} ],
+                "ManagedPolicyArns": [
+                    {"Ref": "LetsLambdaManagedPolicy"},
+                    {"Ref": "LetsLambdaS3WriteManagedPolicy"},
+                    {"Ref": "LetsLambdaS3ReadManagedPolicy"},
+                    {"Ref": "LetsLambdaKmsKeyManagedPolicy"}
```

Comment:
I am in doubt if this is the good approach. Generally, we are creating resources in the User's environment, that can be used for many other things.

A cleaner approach would be to have ONE managed policy with multiple statements in order to preserve the split of Actions + Resources.